### DISCUSSION TITLE
V 0.13  Device Name

### DISCUSSION THREAD

**Discussion by AnKloss** [Louisvdw/dbus-serialbattery#242] [General]
Hello,
I use V 0.13 on Pi4 rev1.5 venus os 2.91.
I cannot rename the JK Bms, it shows 'SerialBattery(jkbms) v0.13 in the main. ( I have 3 JKbms and they all show the same name)
In the folder Device under Product ID missing the field' Name",  
Can we have it back?

mfg
Andreas

**Comment by Louisvdw** [comment]
The custom name option was added and should be available in the nect beta build. For now you can rename, but on a reboot it will be lost.

**Comment by centro-max** [comment]
@Louisvdw 
Where or how can i rename the driver?

**Comment by Louisvdw** [comment]
It will be in the next beta that is released.

**Comment by mr-manuel** [comment]
Check also https://github.com/Louisvdw/dbus-serialbattery/issues/100
